PushNotifications.md

```diff
@@ -326,6 +332,12 @@ Users can listen to messages in each platform using the native message listeners
 
 Take a look at the example app platform specific code to handle messages. For iOS, this is `AppDelegate.m`, and in Android, it is `PushMessagingService.java`. For further help on implementing the Platform specific message handlers, see "On Android" and "On iOS" sections on [Push Notifications - Device activation and subscription](https://ably.com/documentation/general/push/activate-subscribe).
 
+### Additional considerations and resources
+- For tips on how best to use push messaging on Android, read [Notifying your users with FCM](https://android-developers.googleblog.com/2018/09/notifying-your-users-with-fcm.html). For example:
```

Comment:
Are we saying these are best practices (i.e. you _should_ do them) or, as I suspect, tips (i.e. you _could_ do them)? If the latter, then perhaps rephrase to:

```suggestion
- For more tips on using push messaging on Android, read [Notifying your users with FCM](https://android-developers.googleblog.com/2018/09/notifying-your-users-with-fcm.html). For example:
```